### PR TITLE
feat: add sign in and sign up forms

### PR DESCRIPTION
**Pull request by tkachenya41**

### CODE REVIEW

File: src/components/features/InputField/InputField.module.scss [hunk 1]
```diff
@@ -0,0 +1,19 @@
+.container {
+  padding: 1rem 0;
```
Comment (ak-krivonosov) [inline src/components/features/InputField/InputField.module.scss:2]: Please add an empty line after this

File: src/components/features/InputField/InputField.module.scss [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+  input {
+    appearance: none;
+    display: block;
+    line-height: 1rem;
+    padding: 1rem 1.2rem;
+    border-radius: 0.1rem;
+    margin: 0;
+    border: 1px solid var(--primary-active-color);
+    width: 100%;
```
Comment (ak-krivonosov) [inline src/components/features/InputField/InputField.module.scss:18]: Please provide some styles for the focus and disabled states

Comment (ak-krivonosov) [reply src/components/features/InputField/InputField.module.scss:18]: And create an error modifer

File: src/components/features/SignInForm/SignInForm.tsx [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+  const [password, setPassword] = useState("");
+  return (
+    <form className={Style.container}>
+      <InputField
+        label="Email"
+        type="email"
+        placeholder="Enter email"
+        id="email"
+        value={email}
+        onChange={({ target: { value } }) => setEmail(value)}
```
Comment (smillepack) [inline src/components/features/SignInForm/SignInForm.tsx:18]: Please add an email validation and if there is an error change the status of the input

File: src/components/features/SignUpForm/SignUpForm.tsx [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+
+export function SignUpForm() {
+  const [formState, setFormState] = useState<FormState>(getDefaultFormValues);
+  const updateFormValues = useCallback((newFormValue: Partial<FormState>) => {
+    setFormState((previousFields) => ({ ...previousFields, ...newFormValue }));
+  }, []);
+
+  return (
+    <form className={Style.container}>
+      {formSchema.map((field) => (
```
Comment (ak-krivonosov) [inline src/components/features/SignUpForm/SignUpForm.tsx]: Please add validations for password, username and email. If there are errors change the status of the input to show which input is incorrect and show messages that describe the reason of the error. Also if there is an error please disabled the submit button

File: src/components/features/SignInForm/SignInForm.tsx [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+export const SignInForm = () => {
+  const [email, setEmail] = useState("");
+  const [password, setPassword] = useState("");
+  return (
+    <form className={Style.container}>
+      <InputField
+        label="Email"
+        type="email"
+        placeholder="Enter email"
+        id="email"
```
Comment (ak-krivonosov) [inline src/components/features/SignInForm/SignInForm.tsx]: If a user type wrong data and submit it, please change inputs statuses and display some info

File: src/app/sign-in/page.tsx [hunk 6]
```diff
@@ -0,0 +1,5 @@
+import { SignInForm } from "@/components/features/SignInForm/SignInForm";
+
+export default function SignInPage() {
+  return <SignInForm />;
```
Comment (ak-krivonosov) [inline src/app/sign-in/page.tsx]: Please remove the SignInForm and put everything here in the page

File: src/app/sign-up/page.tsx [hunk 7]
```diff
@@ -0,0 +1,5 @@
+import { SignUpForm } from "@/components/features/SignUpForm/SignUpForm";
+
+export default function SignUpPage() {
+  return <SignUpForm />;
```
Comment (ak-krivonosov) [inline src/app/sign-up/page.tsx]: Please remove the SignUpForm and put everything here in the page

**Review by ak-krivonosov — CHANGES_REQUESTED** [review]
Good luck

**Pull request merged**